pgmpy/readwrite/ProbModelXML.py

```diff
@@ -518,6 +536,14 @@ def add_probnet_additionalconstraints(self, constraint):
             argument_value = argument.attrib['value']
             self.probnet['AdditionalConstraints'][constraint_name][argument_name] = argument_value
 
+    def add_criterion(self, criterion):
```

Comment:
Add documentation to this method. What does the input parameters mean and what should be there type?